worker/raft/rafttransport/shim.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package rafttransport
+
+import "gopkg.in/juju/worker.v1"
+
+func NewWorkerShim(config Config) (worker.Worker, error) {
+	return NewWorker(config)
+}
```

Comment:
If you can just pass config in to NewWorker, why do you need NewWorkerShim? Because the return value needs to be  the interface type?